consensus/consensus.go

```diff
@@ -30,8 +30,18 @@ type Consenter interface {
 
 // Inquirer is used to retrieve info about the validating network
 type Inquirer interface {
-	GetNetworkInfo() (self *pb.PeerEndpoint, network []*pb.PeerEndpoint, err error)
-	GetNetworkHandles() (self *pb.PeerID, network []*pb.PeerID, err error)
+	GetOwnID() (id uint64)
```

Comment:
What do all of these interfaces do?  I have a suspicion that we don't need that many.